### ISSUE TITLE
Cassandra fails to start with "Cassandra 2.0 and later require Java 7u25 or later"

### ISSUE THREAD

**Issue by ArnaudBrousseau** [Yelp/casper#10]
Here's the output for `make itest` on my macbook:
```
=> make itest
....
bin/docker-compose-1.19.0 -f itest/docker-compose.yml up -d spectre backend cassandra Creating itest_syslog_1    ... done
Creating itest_cassandra_1 ... done
Creating itest_backend_1   ... done
Creating itest_cassandra_1 ...
Creating itest_spectre_1   ... done
bin/docker-compose-1.19.0 -f itest/docker-compose.yml exec -T cassandra /opt/setup.sh
ERROR: No container found for cassandra_1
make: *** [run-itest] Error 1
```

Looking into the cassandra image, it seems like the problem is cassandra refusing to start:
```
=> bin/docker-compose-1.19.0 -f itest/docker-compose.yml up cassandra
Starting itest_cassandra_1 ... done
Attaching to itest_cassandra_1
cassandra_1  | Cassandra 2.0 and later require Java 7u25 or later.
itest_cassandra_1 exited with code 1
```

...and that's probably because java is busted inside of docker:
```
=> bin/docker-compose-1.19.0 -f itest/docker-compose.yml images
    Container            Repository         Tag       Image Id      Size
-------------------------------------------------------------------------
itest_backend_1     itest_backend          latest   35ad69bdd1ab   173 MB
itest_cassandra_1   itest_cassandra        latest   909b2b4d803d   577 MB
itest_spectre_1     spectre-dev-abrousse   latest   f11d204004e9   484 MB
itest_syslog_1      itest_syslog           latest   566da6e9c7a1   154 MB
=> docker run -ti  909b2b4d803d bash
dckruser@2c86d8724510:/$ java -version
/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/rt.jar: invalid LOC header (bad signature)
Error occurred during initialization of VM
java.lang.NoClassDefFoundError: java/lang/ref/Reference$1
	at java.lang.ref.Reference.<clinit>(Reference.java:235)
```

**Comment by ArnaudBrousseau** [comment]
I think this was just a temporary networking issue corrupting the java installer. Running `bin/docker-compose-1.19.0 -f itest/docker-compose.yml build --no-cache` fixed the problem (`bin/docker-compose-1.19.0 -f itest/docker-compose.yml exec -T cassandra /opt/setup.sh` succeeds)

/me shrugs

**Comment by drolando** [comment]
yeah, this should work fine on your laptop since it happens inside docker
